.github/workflows/tests.yml

```diff
@@ -0,0 +1,175 @@
+name: unit tests
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - release/*
+      - support/*
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+      - develop
+      - release/*
+      - support/*
```

Comment:
Your should not filter branches on `pull_request` event. All pull requests should trigger checks.
```suggestion
```